src/main/java/io/strimzi/kafka/bridge/mqtt/utils/MappingRule.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.mqtt.utils;
+
+import java.util.HashMap;
+
+/**
+ * Represents a Mapping Rule in the Topic Mapping Rules(TOMAR).
+ *
+ */
```

Comment:
Might be good to include an example in the class comment up here of what a user supplied MappingRule would look like

Reply:
@kyguy I agree.